TypeCobol.Analysis.Test/DfaCallPgmReport.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System.Collections.Generic;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using TypeCobol.Compiler;
+using TypeCobol.Compiler.Symbols;
+using System.IO;
+using TypeCobol.Analysis.Dfa;
+using TypeCobol.Analysis.Graph;
+using TypeCobol.Analysis.Report;
+using TypeCobol.Compiler.Nodes;
+
+using static TypeCobol.Analysis.Test.CfgTestUtils;
+
+namespace TypeCobol.Analysis.Test
+{
+    /// <summary>
+    /// Dfa test for call module report
+    /// </summary>
+    [TestClass]
+    public class DfaCallPgmReport
```

Comment:
These 2 tests are duplicate with the 2 in CLI, should we keep them ?

Reply:
Yes, because theses are for CLI using the -cfg WithDfa and -zcr options in the command line arguments.

Reply:
On one hand the CLI tests are more complete because they also cover the new options, on the other hand the tests in TypeCobol.Analysis are easier to debug because they run in the same process...
I'm ok to keep them both but as they are duplicates, we'll have to keep in mind that changes made on one have to be copied on the other.

Reply:
Yes but anyway they still to be different tests.